packages/webapp/src/webview/ui/components/Header/Filters/Filters.tsx

```diff
@@ -210,50 +226,47 @@ export function Filters() {
-                    className="dialog-filter"
-                    dialogContentProps={{ title: filterType[filter].title }}
-                    isOpen={filterType[filter].visibility}
-                    isBlocking={true}
-                    acceptButtonText={'Apply Filter'}
-                    cancelButtonText={'Cancel'}
-                    styles={{ main }}
-                    onAccept={() => filterType[filter].apply()}
-                    onCancel={resetFilter}
-                    onDismiss={resetFilter}>
-                    <VSCodeTextField
-                        style={{ width: '100%' }}
-                        value={query}
-                        onInput={searchFilter}
-                        readOnly={appState.loading}
-                        placeholder="Search"
-                        id="dialog-filter-field"></VSCodeTextField>
-                    <ul className="filter-list">{filterType[filter].listItems}</ul>
-                </UIDialog>
+                <FocusZone
+                    direction={FocusZoneDirection.horizontal}
+                    isCircularNavigation={true}
+                    style={{ display: 'flex' }}>
+                    <UIIconButton
+                        id="filter-products"
+                        iconProps={{ iconName: UiIcons.Table }}
+                        onClick={() => toggleFilters(PRODUCTS)}
+                        disabled={appState.guidedAnswerTreeSearchResult.productFilters.length === 0}
+                        className={`filter-button ${selectedProductFilters.length > 0 ? 'filter-button-selected' : ''}`}
+                        primary
+                        title="Filter Products"></UIIconButton>
+                    <UIIconButton
+                        id="filter-components"
+                        iconProps={{ iconName: UiIcons.IdTag }}
+                        onClick={() => toggleFilters(COMPONENTS)}
+                        disabled={appState.guidedAnswerTreeSearchResult.componentFilters.length === 0}
+                        primary
+                        title="Filter Components"
+                        className={`filter-button ${
+                            selectedComponentFilters.length > 0 ? 'filter-button-selected' : ''
+                        }`}></UIIconButton>
+                    <UIDialog
+                        className="dialog-filter"
+                        dialogContentProps={{ title: filterType[filter].title }}
+                        isOpen={filterType[filter].visibility}
+                        isBlocking={true}
+                        acceptButtonText={'Apply Filter'}
+                        cancelButtonText={'Cancel'}
+                        styles={{ main }}
+                        onAccept={() => filterType[filter].apply()}
+                        onCancel={resetFilter}
+                        onDismiss={resetFilter}>
+                        <UITextInput placeholder="Search!" value={query} onChange={searchFilter} />
```

Comment:
@unseen1980, was the change of the placeholder from **Search** to **Search!** intentionaly?

Reply:
Of course not, thanks for spotting it. I should spot working at night 😞